sim/warrior/shockwave.go

```diff
@@ -0,0 +1,60 @@
+package warrior
+
+import (
+	"time"
+
+	"github.com/wowsims/wotlk/sim/core"
+	"github.com/wowsims/wotlk/sim/core/stats"
+)
+
+func (warrior *Warrior) registerShockwaveSpell() {
+	cost := 15.0 - float64(warrior.Talents.FocusedRage)
+	refundAmount := cost * 0.8
+	cd := 20 * time.Second
```

Comment:
Might as well drop this in
`	cd := 20 - core.TernaryDuration(warrior.HasMajorGlyph(proto.WarriorMajorGlyph_GlyphOfShockwave), 3*time.Second, 0)
`